### PR TITLE
feat(wallet): disable complex routing

### LINKED ISSUE #20869: Wallet: Disable complex routing
Currently the app does not have the infrastructure to handle complex routing, it brings in many secondary issues and is outright complex to a user.
Since we will be moving away from this by 2.31 it makes sense to block it for the 2.30 release as it is a simple fix to do and will reduce some user confusion if they end up making a multichain transaction.

It can be blocked in 
src/status_im/contexts/wallet/send/events.cljs
Screenshot 2024-07-24 at 12 37 23
by updating the `best` route to be `nil` if it is a multichain route.

### LINKED ISSUE #20895: Token value is sent with 0 in multi-routing transaction confirmation
### Steps to Reproduce:
1. Restore a user with available assets on at least 3 networks.
2. Build the multi-route.
3. Confirm the transaction.

### Actual Result:
Only one transaction is sent with the correct value, visible in etherscan.
Other networks send transactions with 0 value, leading to users paying fees for empty transactions.

https://github.com/user-attachments/assets/9009e766-3ea5-496c-a522-8cfaba7b29d3

Example: After confirming a multi-route transaction:
- L1 1.9 USDC ->  is sent successfully. [etherscan](https://etherscan.io/tx/0xf6fb9c7b0c4de613b4b5dc8e4c0a6e55aff413fc3ea9d760f57e39ea134a1392)
- L2 ARB 2.2 USDC ->  is sent with 0 value. [etherscan](https://arbiscan.io/tx/0x0da402b6b6fd5eceea5f90a07ce7c8207e4ee782ae955677a2a99568f175add8)
- L2 OPT 1.2 USDC -> is sent with 0 value. [etherscan](https://optimistic.etherscan.io/tx/0x528088cc08dc36909405130914b34bfb068f9c3f870261d682f0e7fef58c702d)

### Expected Result:
All transactions should be sent with the correct value across all networks involved in the multi-route. OR we should disable multi routing transactions

### Logs:
[Status-debug-logs.zip](https://github.com/user-attachments/files/16393161/Status-debug-logs.zip)

### ENV:
Nightly 27 Jul

### PR DESCRIPTION
**Pull request by briansztamfater**
fixes #20869 
fixes #20895

### Summary

This PR disables routes that involves more than 1 transactions. If best prop of the suggested routes response contains more than 1 element, we replace it with an empty vector.

Linked issues are different, but now that we are disabling multichain routes, expected behavior would be the same for both issues.

#### Platforms

- Android
- iOS

##### Functional

- wallet / transactions

### Steps to test

- Restore a user with available assets on at least 3 networks.
- Try to build a multi-route.
- Expected message should be "No routes found"

status: ready

### Risk

Described potential risks and worst case scenarios.

Tick **one**:
- [ ] Low risk: 2 devs MUST perform testing as specified above and attach their results as comments to this PR **before** merging.
- [x] High risk: QA team MUST perform additional testing in the specified affected areas **before** merging.

### CODE REVIEW

File: src/status_im/contexts/wallet/send/events.cljs [hunk 1]
```diff
@@ -504,6 +504,7 @@
      (let [suggested-routes-new-data (data-store/rpc->suggested-routes data)
            suggested-routes          (-> suggested-routes-new-data
                                          first
+                                         (update :best (fn [best] (if (> (count best) 1) [] best)))
```
Comment (J-Son89) [inline src/status_im/contexts/wallet/send/events.cljs:508]: I think we should add a comment here to explain this very random behaviour :)

**Review by J-Son89 — APPROVED** [review]
nice one @briansztamfater! 🙌

File: src/status_im/contexts/wallet/send/events.cljs (hunk 1 above)
Comment (briansztamfater) [reply src/status_im/contexts/wallet/send/events.cljs:508]: Added!

**Review by BalogunofAfrica — APPROVED** [review]

**Comment by J-Son89** [comment]
tested and working as expected for me 👌

**Comment by VolodLytvynenko** [comment]
hi @briansztamfater thank you for PR. Ready to be merged

File: src/status_im/contexts/wallet/send/events.cljs (hunk 1 above)
Comment (ulisesmac) [reply src/status_im/contexts/wallet/send/events.cljs:508]: What will the UI show if the multichain route is removed?

**Review by ulisesmac — APPROVED** [review]

File: src/status_im/contexts/wallet/send/events.cljs (hunk 1 above)
Comment (briansztamfater) [reply src/status_im/contexts/wallet/send/events.cljs:508]: In this case it would end to a `No routes found` case

**Pull request merged**